packages/codemirror-graphql/.npmignore

```diff
@@ -0,0 +1,4 @@
+babel.config.js
+.gitignore
+resources
```

Comment:
By not publishing `resources` here, the reference at https://github.com/graphql/graphiql/blob/98be6f5df0f2e01054df51377949d401c5e6ad01/packages/graphiql/src/components/ResultViewer.js#L45

is no longer valid. Either this line or that line should be removed.

Reply:
I’m confused? that import is from codemirror-graphql, which still exports that module. are you having issues? this PR was merged

Reply:
if what you are saying is true, then the results viewer would fail to highlight jaon here in the current build of master:

https://graphiql-test.netlify.com

Reply:
Sorry, I got `resources` and `results` confused so this isn't the direct issue. Regardless, there's still an issue with the latest `codemirror-graphql` package because the `results` folder, referenced in the line above, does not exist.

```sh
~ npm install codemirror-graphql@0.11.5
~ ls node_modules/codemirror-graphql/results
ls: node_modules/codemirror-graphql/results: No such file or directory
```

This broken link is causing my build to fail, so I've resorted back to `graphiql@0.17.2` and manually pinning `codemirror-graphql@0.11.4` (which does publish this folder) for now.

Reply:
@caseywebdev try now with `0.11.6`

Reply:
https://unpkg.com/browse/codemirror-graphql@0.11.6/ my bad! we should be doing pre-releases for all these build changes.

Reply:
Working again 👍🏻 Thanks for the quick turnaround!